app/blueprints/assessments/routes.py

```diff
@@ -680,6 +683,83 @@ def application(application_id):
     sub_criteria_status_completed = all_status_completed(state)
     form = AssessmentCompleteForm()
     associated_tags = get_associated_tags_for_application(application_id)
+    add_comment_argument = request.args.get("add_comment") == "1"
+    edit_comment_argument = request.args.get("edit_comment") == "1"
+    comment_form = CommentsForm()
+
+    if add_comment_argument and comment_form.validate_on_submit():
+        comment = comment_form.comment.data
+        submit_comment(
+            comment=comment,
+            application_id=application_id,
+            sub_criteria_id="",
```

Comment:
Might be worth adding a comment in the code here noting why sub criteria is blank?
```suggestion
            sub_criteria_id="",  # No subcriteria id indicates it belongs to the entire application.
```

Or something like that?